lib/serializer.js

```diff
@@ -77,11 +78,18 @@ module.exports = class CustomSerializer extends Serializer {
 
     _serializeSlot(node) {
         const slotName = node.attribs.name;
-        if (this.slotMap.has(slotName)) {
+        if (slotName && this.slotMap.has(slotName)) {
             const slots = this.slotMap.get(slotName);
             if (slots) {
                 slots.forEach(this._serializeNode);
             }
+        } else {
+            // Unnamed slot behaviour
+            const defaultSlots = this.slotMap.get('default');
```

Comment:
[Here](https://github.com/zalando/tailor/blob/master/lib/transform.js#L33) i am grouping them together to default slots if they don't have a slot name